programs/clearing_house/src/controller/lp.rs

```diff
@@ -0,0 +1,52 @@
+use crate::error::ClearingHouseResult;
+use crate::math::casting::cast_to_u128;
+use crate::math::lp::get_lp_metrics;
+use crate::math::lp::SettleResult;
+use crate::math_error;
+use crate::state::market::AMM;
+use crate::MarketPosition;
+use solana_program::msg;
+
+pub fn settle_lp_position(
```

Comment:
i think we want to consider, whether like settle_funding_payment, if we just call settle_lp_position for every position in the beginning of every ix